common/src/main/java/com/redhat/cloud/notifications/models/CamelProperties.java

```diff
@@ -21,6 +22,7 @@
 public class CamelProperties extends EndpointProperties implements SourcesSecretable {
 
     @NotNull
+    @ValidNonPrivateUrl
```

Comment:
Could you please test this annotation with our REST API? Here's an example of something similar:
https://github.com/RedHatInsights/notifications-backend/blob/415d7ab35236b52fcba33bd9844f251830525d3f/backend/src/test/java/com/redhat/cloud/notifications/routers/EndpointResourceTest.java#L302-L325
Both `CamelProperties` and `WebhookProperties` will need to be tested that way.

Reply:
Added a couple of tests!